microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
    limitations under the License.  -->
 # Optimize website speed using Lighthouse
 
-This tutorial teaches you how to use DevTools to find ways to make your websites load faster.
+This tutorial teaches you how to use Lighthouse, and other tools, in DevTools to find ways to make your websites load faster.
 
-<!-- terminology hit counts as of 1/26/2022:
-audit: 93 hits
-lighthouse: 9 -->
-This tutorial uses the **Lighthouse** tool.  The **Lighthouse** tool provides links to content hosted on third-party websites.  Microsoft is not responsible for and has no control over the content of these sites and any data that may be collected.
-
-The **Lighthouse** tool was previously called the **Audits** tool.  "Audits panel" means the same thing as "Lighthouse tool".
+The **Lighthouse** tool provides links to content hosted on third-party websites.  Microsoft isn't responsible for and has no control over the content of these sites and any data that may be collected.
 
 
 <!-- ====================================================================== -->
 ## Prerequisites
 
-*  You should have basic web development experience, similar to what is taught in this [Introduction to Web Development class](https://www.coursera.org/learn/web-development#syllabus).
+* You should have some web development experience, similar to what is taught in this [Introduction to Web Development class](https://www.coursera.org/learn/web-development#syllabus).
```

Comment:
i'm uneasy with any statements like "you must not be a beginner".   actually beginners are welcome to jump into every page on the web, and if they happen to want a basic instruction page, we offer that.

It is possible to write this tut for beginners, so i'd aim to support that, while still offering a link for additional optional info.

Right now the sentence effectively says "You should not use this article, if."  "This article is not for you."  It is possible to have positive invitation framing instead, so recomm impl'g that.